pkg/chartverifier/checks/charttesting.go

```diff
@@ -16,6 +17,31 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+// Versioner provides OpenShift version
+type Versioner func() (string, error)
+
+var getVersion = func() (string, error) {
```

Comment:
This could be just a function definition: `func getVersion() (string, error) {...`

Reply:
Done